### PR TITLE
Configuring Kafka & code for user details - Review required

### PR DESCRIPTION
**Pull request by shubhangis91**
Configuring Kafka Code for UserManagement service #11

### CODE REVIEW

**Comment by adithyaselv** [comment]
Can you also use a containerised mysql server or mongoDB for simplicity ?

**Comment by shubhangis91** [comment]
MySQL containerization - Done

File: UserManagement/README.md [hunk 1]
```diff
@@ -1,32 +0,0 @@
-### Software requirements
```
Comment (adithyaselv) [inline UserManagement/README.md:1]: Are these instructions not valid anymore ?

File: UserManagement/src/test/user_mgmt [hunk 2]
```diff
@@ -1,83 +0,0 @@
-{
```
Comment (adithyaselv) [inline UserManagement/src/test/user_mgmt]: ^ ditto

**Review by adithyaselv — CHANGES_REQUESTED** [review]
please fix the above

File: UserManagement/README.md (hunk 1 above)
Comment (shubhangis91) [reply UserManagement/README.md:1]: They are valid. Just need to append instructions for Docker as well.

File: UserManagement/src/test/user_mgmt (hunk 2 above)
Comment (shubhangis91) [reply UserManagement/src/test/user_mgmt]: Resolved. Added files.

File: UserManagement/README.md [hunk 3]
```diff
@@ -1,32 +1,113 @@
...
-```sh
-maven package
-```
-
-8. Run the application using Maven with the following command.
-
-```sh
-mvn spring-boot:run
-```
+<<<<<<< HEAD
```
Comment (adithyaselv) [inline UserManagement/README.md]: guess you forget to remove this when resolving merge conflict could you please resolve it ?

**Review by adithyaselv — APPROVED** [review]

**Pull request merged**